index.bs

```diff
@@ -199,8 +199,9 @@ A variety of additional use cases and configurations are also possible, includin
 
 # Conformance # {#conformance}
 
-This specification defines three conformance classes. Each of these classes is specified so that conforming members of the class
-are secure against non-conforming or hostile members of the other classes.
+This specification defines two conformance classes. Each of these classes is specified so that conforming members of the class
+are secure against non-conforming or hostile members of the other classes. The specification recommends a set of 
+[[#rp-operations]] that [=[RP]=] SHOULD perform to get the security benefits offered by this specification. 
```

Comment:
"[=[RPS]=]", right? See https://s3.amazonaws.com/pr-preview/AngeloKai/webauthn/informative.html#conformance to check the final text.